docs/dev/table/streaming/temporal_tables.md

```diff
@@ -78,9 +82,36 @@ The concept of *Temporal Tables* aims to simplify such queries, speed up their e
 
 In the above example `currency` would be a primary key for `RatesHistory` table and `rowtime` would be the timestamp attribute.
 
-In Flink, a temporal table is represented by a *Temporal Table Function*.
+In Flink, this is represented by a *Temporal Table Function*.
+
+On the other hand, we have the requirement to join a changing dimension table which is an external database table.
```

Comment:
Use headings to separate both examples instead of putting them into one long text.

We should avoid `we` in documentation, who is `we` in this context? Instead you could rephrase to:
`On the other hand, some use cases require to join a changing dimension table which is an external database table.` `Let's assume that `LatestRates` is a table (e.g. stored in)`